.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
   - name: "py36-np111"
     env: PY=3.6 NUMPY=1.11
   - name: "py36-np119"
-    env: PY=3.6 NUMPY=1.19
+    env: PY=3.6 NUMPY=1.18
```

Comment:
We probably need to document or fix this. (Or drop support for Python 3.6.)

What do you think @efiring?

Reply:
It turns out we cannot do numpy >1.17 on python 3.6. @efiring are you OK dropping support for Python 3.6 or should we add conditional dependencies (if python 3.6, numpy <=1.17)?

Reply:
This seems to be an issue with the CI system (or conda?). In a "clean" python 3.6.10 pyenv, I was able to `python setup.py install` and it installed numpy 1.19.0, built, and tested successfully.

Reply:
Probably conda's numpy. We would need to rebuilt it. However, conda-forge already dropped support for 3.6, so that is not happening. If you look at the support table at https://numpy.org/neps/nep-0029-deprecation_policy.html we are "OK-ish" dropping support, which means no testing or bugfixing, but the choice to build it is still out there for those who want to do so.

Reply:
No worries, looks like this was sorted out before I managed to push "comment".